wordpress-mobile/WordPress-iOS
# Signup Epilogue: update Done button

Ref #13939 

This updates the `Done` button on the Signup Epilogue to the new design. Except the button color. That's still pink to match the rest of the flow.

Technically:

Previously, `NUXButtonViewController` was used to display the button. This added most of the style that we no longer want. So I replaced it with a simple `FancyButton` (which is what the _Login_ Epilogue uses).

Visually:
- The button title is now `Done` (it was `Continue`).
- The area around the button now matches the table view color.
- The shadow between the button and the table has been removed.

| ![light](https://user-images.githubusercontent.com/1816888/80549571-e0473580-897a-11ea-91a8-b742d49a7e15.png) | ![dark](https://user-images.githubusercontent.com/1816888/80549583-e6d5ad00-897a-11ea-8d70-539134edd016.png) |
|--------|-------|

---
To test:
- Signup. (See below if you don't want to actually signup)
- Verify the `Done` button looks like that on #13939 or in Zeplin.

---
To show the Signup Epilogue without actually signing up:

- In WPiOS `Podfile`, point `WordPressAuthenticator` to your local Auth.
- In `LoginViewController`, change `showLoginEpilogue` to show the signup epilogue:

```
func showLoginEpilogue(for credentials: AuthenticatorCredentials) {
    guard let navigationController = navigationController else {
        fatalError()
    }
    
    showSignupEpilogue(for: credentials)
    
//        authenticationDelegate.presentLoginEpilogue(in: navigationController, for: credentials) { [weak self] in
//            self?.dismissBlock?(false)
//        }
}
```

- _Login_ with an existing WP account, and the _signup_ epilogue will be displayed.

---
PR submission checklist:
